source/Outfit.cpp

```diff
@@ -204,6 +204,13 @@ void Outfit::Load(const DataNode &node)
 			category = child.Token(1);
 		else if(child.Token(0) == "plural" && child.Size() >= 2)
 			pluralName = child.Token(1);
+		else if(child.Token(0) == "escape ship")
```

Comment:
I would prefer to use the language of "escape pod" as was done in #5977. People know what you're talking about when you say "escape pod," and even if an "escape pod" doesn't necessarily need to be a "pod," I think it's easier to explain "Your ship's escape pod can actually be any ship," than it is to say "Your ship's escape ship is the same thing as an escape pod, but we call it an escape ship because it can be any ship."